packages/driver/cypress/integration/commands/fixtures_spec.js

```diff
@@ -101,21 +101,14 @@ describe('src/cy/commands/fixtures', () => {
         return null
       })
 
-      it('throws if fixturesFolder is set to false', {
-        fixturesFolder: false,
```

Comment:
Just curious, how is this error handled when configured via a suite or test configuration like this previous test was doing?

Reply:
@emilyrohrbough It fails the same way (see screenshot below), I just moved it into the test using `Cypress.Config()` so I would be able to catch the error with `cy.on('fail')`
<img width="554" alt="Screenshot 2021-11-16 at 10 50 23 AM" src="https://user-images.githubusercontent.com/27690333/142018599-375a3072-a8a7-42a4-8080-27d61ef7a403.png">

Reply:
Nice! It appears that the correct line number is displaying. It does look like the error message says "Cypress.config() cannot be called...", but this test was setting test configuration via the test options, not Cypress.config(). We might want to update the working depending how the configuration was set.